requirements.txt

```diff
@@ -0,0 +1 @@
+check-jsonschema
```

Comment:
What's this for?

Reply:
It's the `pip` package containing the tool we're using to validate the input file.

I found that the `pip` caching thing didn't work if I just did `pip install check-jsonschema`